lib/Backends/OpenCL/OpenCLDeviceManager.cpp

```diff
@@ -50,6 +50,17 @@ DeviceManager *createOCLDeviceManager(std::unique_ptr<DeviceConfig> config) {
 } // namespace runtime
 } // namespace glow
 
+llvm::Error parseInputAsUnsigned(std::string input, unsigned &output) {
```

Comment:
nit: I think this would be cleaner as an `llvm::Expected<unsigned>`, and then use `ASSIGN_VALUE_OR_RETURN_ERR()` at the callsite, instead of passing the reference to the output.